libraries/plugins/peerplays_sidechain/peerplays_sidechain_plugin.cpp

```diff
@@ -343,6 +345,40 @@ void peerplays_sidechain_plugin_impl::recreate_primary_wallet()
 }
 
 void peerplays_sidechain_plugin_impl::process_deposits() {
+
+   // Account who issues tokens to the user who made deposit
+   account_id_type pay_from = GRAPHENE_NULL_ACCOUNT;
+   const auto& account_idx = plugin.database().get_index_type<account_index>().indices().get<by_name>();
+   const auto& account_itr = account_idx.find("nathan");
+   if (account_itr != account_idx.end())
+      pay_from = (*account_itr).id;
```

Comment:
shouldn't pay_from be from a pool controlled by all SONs?

Reply:
Yes, this is a shortcut so I can confirm that transfer proposals gets approved, and transfer executed. For full featured SON controlled account, we need something like a primary wallet for SONs. Thats something we still need to think about.